helm/alfresco-identity-service/alfresco-realm.json

```diff
@@ -980,6 +980,47 @@
       "useTemplateScope": false,
       "useTemplateMappers": false
     },
+    {
+      "id": "da2b61de-bd1d-47d1-8106-2fb93cd6b0f2",
+      "clientId": "desktop-sync",
```

Comment:
We'll want to standardize on an `alfresco-` prefix for our clients to distinguish from clients that customers might add to the same realm.

Reply:
Added the prefix, thank you!